src/message_replies/reportChannelPoints.js

```diff
@@ -28,13 +29,45 @@ const reportChannelPoints = async (interaction) => {
     );
   }
 
+  // if the caller is not in the top 8, go get their point data and append it to the end of the list
+  if (
+    !arrOfValidUsers.filter((validUser) => validUser.user.id === callerId)
+      .length > 0
+  ) {
+    const callerData = await BuddiesModel.getUser(callerId).catch((err) =>
+      console.log(err)
+    );
+
+    const { username, validUser, user } = await isUserValid(
+      interaction,
+      callerId
+    );
+
+    validUser
+      ? arrOfValidUsers.push({
+          username,
+          points: callerData.pointsReceived.length,
+          user,
+        })
+      : null;
+  }
+
   const caller = interaction.author.username;
   const channelName = interaction.channel.name;
   const userListHTML = leaderBoardBody(channelName, caller, arrOfValidUsers);
   const body = generateBody(userListHTML, styles);
 
-  const img = await imgFromHtmlGenerator(body);
-  interaction.reply({ files: [{ attachment: img }] });
+  try {
+    const img = await imgFromHtmlGenerator(body, interaction);
+    if (!(img instanceof Error)) {
+      interaction.reply({ files: [{ attachment: img }] });
+    }
+  } catch (err) {
+    console.log(err);
+    interaction.reply(
+      "Oof, sorry. I couldn't figure out who the leader is. Try again."
+    );
+  }
```

Comment:
again, part of other commit. i'm not sure if will have any effect, but I thought that I should probably point that out